spec/views/defendants/_offences.html.haml_spec.rb

```diff
@@ -11,6 +11,24 @@
       assign(:defendant, defendant)
     end
 
+    context 'when offence has a start date' do
+      before { allow(offence).to receive(:start_date).and_return('2023-01-01') }
+
+      it 'displays offence start date' do
+        is_expected.to have_css('.govuk-table__cell',
+                                text: %r{01/01/2023})
+      end
+    end
+
+    context 'when offence has no start date' do
+      before { allow(offence).to receive(:start_date).and_return(nil) }
+
+      it 'displays no offence start date' do
+        is_expected.to have_css('.govuk-table__cell',
+                                text: /Not available/)
```

Comment:
"Not available" string may be visible in other part of the UI,
in that case this check is too generic.

(but I am not sure how easily we can identify that part of the html)